### PR TITLE
ACS-2499 Delete company release build stage.

### PR DESCRIPTION
**Pull request by tpage-alfresco**
This never uploaded any software, and so was not particularly useful. The software and license report should be
uploaded to S3 as part of the transform-service build.

Also skip uploading anything to staging bucket.

### CODE REVIEW

**Review by alandavis — APPROVED** [review]
LGTM - I was just about to create the same PR - I think we should let @montgolfiere (Jan) also approve

File: docs/build-and-release.md [hunk 1]
```diff
@@ -5,8 +5,7 @@ The `.travis.yml` config file can be found in the root of the repository.
 
 ## Stages and Jobs
 1. **Build**: Java build with unit and integration tests.
-2. **Release**: Release with artifact deployment to Nexus and AWS Staging bucket.
-3. **Company Release**: Artifact deployment to AWS Release bucket.
+2. **Release**: Release with artifact deployment to Nexus.
```
Comment (montgolfiere) [inline docs/build-and-release.md:8]: Minor: also to DockerHub & Quay.io

**Review by montgolfiere — APPROVED** [review]
LGTM - minor comment added (for readme).

**Review by mpichura — APPROVED** [review]

File: docs/build-and-release.md (hunk 1 above)
Comment (alandavis) [reply docs/build-and-release.md]: Done

**Pull request merged**